test/TableGateway/Feature/FeatureSetTest.php

```diff
@@ -77,4 +78,93 @@ public function testAddFeatureThatFeatureHasTableGatewayButFeatureSetDoesnotHas(
         $featureSet = new FeatureSet;
         $this->assertInstanceOf('Zend\Db\TableGateway\Feature\FeatureSet', $featureSet->addFeature($feature));
     }
+
+    /**
+     * @cover FeatureSet::canCallMagicCall
```

Comment:
Thanks for that. I should have picked that up sooner. Perhaps I should go to bed as it's nearly 2am.